"use strict";(self.webpackChunkGATK_SV=self.webpackChunkGATK_SV||[]).push([[9671],{3905:(e,t,n)=>{n.d(t,{Zo:()=>u,kt:()=>m});var r=n(7294);function a(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function i(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);t&&(r=r.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,r)}return n}function o(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?i(Object(n),!0).forEach((function(t){a(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):i(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function l(e,t){if(null==e)return{};var n,r,a=function(e,t){if(null==e)return{};var n,r,a={},i=Object.keys(e);for(r=0;r<i.length;r++)n=i[r],t.indexOf(n)>=0||(a[n]=e[n]);return a}(e,t);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(r=0;r<i.length;r++)n=i[r],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(a[n]=e[n])}return a}var s=r.createContext({}),c=function(e){var t=r.useContext(s),n=t;return e&&(n="function"==typeof e?e(t):o(o({},t),e)),n},u=function(e){var t=c(e.components);return r.createElement(s.Provider,{value:t},e.children)},p="mdxType",d={inlineCode:"code",wrapper:function(e){var t=e.children;return r.createElement(r.Fragment,{},t)}},f=r.forwardRef((function(e,t){var n=e.components,a=e.mdxType,i=e.originalType,s=e.parentName,u=l(e,["components","mdxType","originalType","parentName"]),p=c(n),f=a,m=p["".concat(s,".").concat(f)]||p[f]||d[f]||i;return n?r.createElement(m,o(o({ref:t},u),{},{components:n})):r.createElement(m,o({ref:t},u))}));function m(e,t){var n=arguments,a=t&&t.mdxType;if("string"==typeof e||a){var i=n.length,o=new Array(i);o[0]=f;var l={};for(var s in t)hasOwnProperty.call(t,s)&&(l[s]=t[s]);l.originalType=e,l[p]="string"==typeof e?e:a,o[1]=l;for(var c=2;c<i;c++)o[c]=n[c];return r.createElement.apply(null,o)}return r.createElement.apply(null,n)}f.displayName="MDXCreateElement"},9881:(e,t,n)=>{n.r(t),n.d(t,{assets:()=>s,contentTitle:()=>o,default:()=>d,frontMatter:()=>i,metadata:()=>l,toc:()=>c});var r=n(7462),a=(n(7294),n(3905));const i={title:"About",description:"GATK-SV",sidebar_position:1},o=void 0,l={unversionedId:"intro",id:"intro",title:"About",description:"GATK-SV",source:"@site/docs/intro.md",sourceDirName:".",slug:"/intro",permalink:"/gatk-sv/docs/intro",draft:!1,editUrl:"https://github.com/broadinstitute/gatk-sv/tree/master/website/docs/intro.md",tags:[],version:"current",sidebarPosition:1,frontMatter:{title:"About",description:"GATK-SV",sidebar_position:1},sidebar:"tutorialSidebar",next:{title:"Getting Started",permalink:"/gatk-sv/docs/category/getting-started"}},s={},c=[{value:"Citation",id:"citation",level:3}],u={toc:c},p="wrapper";function d(e){let{components:t,...n}=e;return(0,a.kt)(p,(0,r.Z)({},u,n,{components:t,mdxType:"MDXLayout"}),(0,a.kt)("p",null,"GATK-SV is a comprehensive, cloud-based ensemble pipeline to capture and annotate all\nclasses of structural variants (SV) from whole genome sequencing (WGS). It can detect\ndeletions, duplications, multi-allelic copy number variants, balanced inversions,\ninsertions, translocations, and a diverse spectrum of complex SV. Briefly, GATK-SV\nmaximizes the sensitivity of SV discovery by harmonizing output from five tools\n(Manta, Wham, cnMOPS, GATK-gCNV, MELT). In order to reduce false positives, raw SV\nare adjudicated and re-genotyped from read evidence considering all potential\nsequencing evidence including anomalous paired-end (PE) reads, split reads (SR) crossing\na breakpoint, normalized read-depth (RD), and B-allele frequencies (BAF). It also\nfully resolves 11 classes of complex SV based on their breakpoint signature. GATK-SV\nhas been designed to be deployed in the Google Cloud Platform via the cromwell\nexecution engine, which allows massively parallel scaling. Further details about\nGATK--SV can be found in ",(0,a.kt)("a",{parentName:"p",href:"https://www.nature.com/articles/s41586-020-2287-8"},"Collins et al. 2020, Nature"),"."),(0,a.kt)("p",null,"A high-level description of GATK-SV is available ",(0,a.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/9022487952155-Structural-variant-SV-discovery"},"here"),"."),(0,a.kt)("h3",{id:"citation"},"Citation"),(0,a.kt)("p",null,"Please cite the following publication:"),(0,a.kt)("ul",null,(0,a.kt)("li",{parentName:"ul"},(0,a.kt)("a",{parentName:"li",href:"https://doi.org/10.1038/s41586-020-2287-8"},'Collins, Brand, et al. 2020. "A structural variation reference for medical and population genetics." Nature 581, 444-451.'))),(0,a.kt)("p",null,"Additional references: "),(0,a.kt)("ul",null,(0,a.kt)("li",{parentName:"ul"},(0,a.kt)("a",{parentName:"li",href:"https://doi.org/10.1038/s41588-018-0107-y"},'Werling et al. 2018. "An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder." Nature genetics 50.5, 727-736.'))))}d.isMDXComponent=!0}}]);